.github/workflows/azure-pipelines.yml

```diff
@@ -0,0 +1,71 @@
+# ASP.NET
+# Build and test ASP.NET projects.
+# Add steps that publish symbols, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/aspnet/build-aspnet-4
+
+trigger:
+- main
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  solution: '**/*.csproj'
+  buildPlatform: 'Any CPU'
+  buildConfiguration: 'Release'
+  agentVersion: '2.6.4.2' #Agent version that is going to be used
+  projectDirectory: '.' #path to one project in your solution
+  webAppName: 'insecure-bank' #your app service name
+  consoleURL: 'http://console-host:8089/hdiv-console-services' #Console address. Set to empty if you do not want to use the console
```

Comment:
Maybe set it empty by default?

Reply:
Actually, we were talking yesterday regarding this value and the agreement was to leave a fake URL with port and "hdiv-console-services" to give an idea of how the real value should look. This is what was also decided for the documentation.